core/txbuilder/finalize_test.go

```diff
@@ -25,6 +25,69 @@ import (
 	"chain/testutil"
 )
 
+func TestSighashCheck(t *testing.T) {
```

Comment:
Not without considerable effort, since this test depends on `account` and `account` depends on `txbuilder`.